### PR TITLE
add network policy for all applications templates

### PR DESCRIPTION
**Pull request by qiliRedHat**
https://issues.redhat.com/browse/OCPQE-8916

In 4.10 network policy is supported in reliability test  https://issues.redhat.com/browse/OCPQE-8911 , but only added network policy to 'nodejs-postgresql-persistent', that means to test network policy, the application template can be only 'nodejs-postgresql-persistent'. This PR will add network policy for more applications.

Instead of adding network policy for all applications. Add a general network policy that works for all application templates. So that there is only one network policy file to be maintain.

### CODE REVIEW

**Comment by qiliRedHat** [comment]
Test:
1.  Add network policy logging to all namespaces to verify network policy is in use.
```
% for ns in $(oc get ns | grep test| awk '{ print $1}'); do oc annotate ns ${ns} k8s.ovn.org/acl-logging='{ "deny": "alert", "allow": "alert" }';done
```
2. Verify successfully on multiple namespaces:
```
% for pod in $(oc get pods -n openshift-ovn-kubernetes -l app=ovnkube-node --no-headers=true | awk '{ print $1 }') ; do
    oc exec -it $pod -c ovn-controller -n openshift-ovn-kubernetes -- tail -1 /var/log/ovn/acl-audit-log.log 
  done
2022-04-11T09:21:13.264Z|41063|acl_log(ovn_pinctrl0)|INFO|name="testuser-10-1_allow-same-namespace_0", verdict=allow, severity=alert: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:81:02:01,dl_dst=0a:58:0a:81:02:c1,nw_src=10.129.2.197,nw_dst=10.129.2.193,nw_tos=0,nw_ecn=0,nw_ttl=63,tp_src=50146,tp_dst=5432,tcp_flags=ack
2022-04-11T09:21:22.928Z|37739|acl_log(ovn_pinctrl1)|INFO|name="testuser-11-1_allow-same-namespace_0", verdict=allow, severity=alert: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:83:00:01,dl_dst=0a:58:0a:83:00:9c,nw_src=10.131.0.182,nw_dst=10.131.0.156,nw_tos=0,nw_ecn=0,nw_ttl=63,tp_src=38178,tp_dst=5432,tcp_flags=ack
2022-04-11T08:28:49.975Z|14455|acl_log(ovn_pinctrl0)|INFO|name="testuser-6-1_allow-same-namespace_0", verdict=allow, severity=alert: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:80:02:01,dl_dst=0a:58:0a:80:02:41,nw_src=10.129.2.222,nw_dst=10.128.2.65,nw_tos=0,nw_ecn=0,nw_ttl=63,tp_src=47266,tp_dst=5432,tcp_flags=ack
```

**Comment by qiliRedHat** [comment]
@mffiedler @paigerube14 PTAL

File: reliability-v2/README.md [hunk 1]
```diff
@@ -221,13 +221,15 @@ The following funcs are supported now:
 | new_project | number of projects | developer | Create n projects for the user|
 | check_all_projects  | N/A | developer | Check projects under the user|
 | new_app  | number of projects | developer | New an app under each project|
-| load_app  | number of projects number of clients | developer | Load an app under each project with a number of clients|
+| load_app  | number of projects  number of clients | developer | Load an app under each project with a number of clients|
+| apply  | number of projects  file location | all | apply a file|
```
Comment (paigerube14) [inline reliability-v2/README.md:225]: looks like there are 2 "apply"s in the list. do both work? Are they used under different sections that we need to section off this list?

File: reliability-v2/config/example_reliability.yaml [hunk 2]
```diff
@@ -44,6 +44,7 @@ reliability:
       tasks:
         - func delete_all_projects # clear all projects
         - func new_project 2 # new 2 projects
+        #- func apply 2 "<path to /reliability-v2/networkpolicy/allow-same-namespace.yaml>" # Apply network policy to 2 projects
```
Comment (paigerube14) [inline reliability-v2/config/example_reliability.yaml:47]: do you want to have this commented out?

File: reliability-v2/README.md (hunk 1 above)
Comment (qiliRedHat) [reply reliability-v2/README.md]: Aha, thanks! my last change was not saved on local. I have fixed this, with some other little changes to README.md.

File: reliability-v2/config/example_reliability.yaml (hunk 2 above)
Comment (qiliRedHat) [reply reliability-v2/config/example_reliability.yaml:48]: @paigerube14 Yeah I intended to do this, as not all the tests needs network policy. I added a comment above this line to explain that this line is valid but only use it when you want to include network policy in the test.

**Pull request merged**